hardware/raspberrypi/bootmodes/gpio.md

```diff
@@ -45,6 +45,6 @@ The various boot modes are attempted in the numerical order of the GPIO lines, i
 
 ## Boot flow
 
-SD0 is the Broadcom SD card / MMC interface. When the boot ROM within the SoC runs, it always connects SD0 to the built-in microSD card slot. On Compute Modules with an eMMC device, SD0 is connected to that; on the Compute Module Lite SD0 is available on the edge connector and connects to the microSD card slot in the CMIO carrier board. SD1 is the Arasan SD card / MMC interface which is also capable of SDIO. All Raspberry Pi models with built-in wifi use SD1 to connect to the wifi chip via SDIO.
+SD0 is the Broadcom SD card / MMC interface. When the boot ROM within the SoC runs, it always connects SD0 to the built-in microSD card slot. On Compute Modules with an eMMC device, SD0 is connected to that; on the Compute Module Lite SD0 is available on the edge connector and connects to the microSD card slot in the CMIO carrier board. SD1 is the Arasan SD card / MMC interface which is also capable of SDIO. All Raspberry Pi models with built-in wireless LAN support use SD1 to connect to the wireless chip via SDIO.
```

Comment:
"All Raspberry Pi models with built-in wireless LAN support use SD1 to connect to the wireless chip via SDIO."

IMHO the "support use" feels a bit weird here (or maybe I'm just reading it wrong?!). I wonder if "All Raspberry Pi models with built-in wireless networking use SD1 to connect to the wireless chip via SDIO." works better?

Reply:
Indeed - that's one of my typos. So:

support use SD1 -> use SD1